frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -60,9 +66,34 @@ const LineAreaBarChart = ({
     chartSize,
   );
 
+  const keys = series
+    .map(singleSeries => {
+      if (singleSeries.seriesKey) {
+        return singleSeries.seriesKey;
+      }
+
+      return formatStaticValue(singleSeries.name, {
+        column: singleSeries.column,
+      });
+    })
+    .filter(isNotNull);
+  const palette = { ...colors, ...instanceColors };
```

Comment:
Actually, there is a `getColor` function, but it isn't compatible with APIs in 
https://github.com/metabase/metabase/blob/cc589cf0f6a3f4dfe8e22aee2c1f21a8a595ddd0/frontend/src/metabase/lib/colors/palette.ts#L83
since that expects a whole color palette to be passed not a function.

That's why I construct the whole palette with user colors instead.